### PR TITLE
Extract NEWS to doc/NEWS.md

### PR DESCRIPTION
**Pull request by pschmitt**
- Move NEWS to doc/NEWS.md
- Update module installation heading

Fixes https://github.com/zdharma-continuum/zinit/issues/35

### CODE REVIEW

File: zinit-install.zsh [hunk 1]
```diff
@@ -1354,7 +1354,7 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || { builtin print -P "${ZINIT
     } elif [[ -n ${ZINIT_SICE[$id_as]} ]] {
         +zinit-message "{error}WARNING:{msg2} Inconsistency #3" \
             "occurred, please report the string: \`{obj}${ZINIT_SICE[$id_as]}{msg2}' to the" \
-            "GitHub issues page: {obj}https://github.com/zdharma-continuum/zinit/issues/{msg2}.{rst}"
```
Comment (alichtman) [inline zinit-install.zsh:1357]: This should be reversed.

**Comment by alichtman** [comment]
Cleaning up the README is great. Minor naming regression to fix

**Review by alichtman — CHANGES_REQUESTED** [review]

File: zinit-install.zsh (hunk 1 above)
Comment (pschmitt) [reply zinit-install.zsh:1357]: Woops..! On it.

Comment (pschmitt) [reply zinit-install.zsh]: Done: 95d1a2fb44e416dbebed744fb5c7b8bbf61184e5

**Review by alichtman — APPROVED** [review]

**Comment by pschmitt** [comment]
Sweet!

**Pull request merged**